docs/source/guides/userguide.rst

```diff
@@ -20,8 +21,73 @@ then a :obj:`~pyDeltaRCM.deltaRCM_driver.pyDeltaRCM` model instance initialized
 Multiple parameters can be specified line by line.
 
 Default values are substituted for any parameter not explicitly given in the ``input_file`` ``.yaml`` file.
```

Comment:
The convention specified here is a break from what precedes the code block, `.yml` switches to `.yaml` and the name of the file is separated from the extension.

Reply:
:+1:  thanks